workflows/rules/normalize.smk

```diff
@@ -1,28 +1,19 @@
-"""
-rule module: normalize.smk
-
-Utlizes pycytominer's normalization module:
-https://github.com/cytomining/pycytominer/blob/c90438fd7c11ad8b1689c21db16dab1a5280de6c/pycytominer/normalize.py
-
-Normalizing single-cell or aggregate features. Current default normalization
-method is `standardize` other methods include:
-
-
-parameters
-----------
-input
-    single-cell or aggregated profiles
+rule normalize:
+    """
+    Normalizing single-cell or aggregate features. Current default normalization
+    method is `standardize` other methods include:
 
-output
-    normalized single-cell or aggregate dataset.
+    Utlizes pycytominer's normalization module:
+    https://github.com/cytomining/pycytominer/blob/c90438fd7c11ad8b1689c21db16dab1a5280de6c/pycytominer/normalize.py
 
-Output
-------
-    Generates an annotated profile stored in the `results/` directory
-"""
+    :input profile: feature selected profiles.
```

Comment:
Same comment as in the documentation file.

Reply:
Answered above